homework.js

```diff
@@ -0,0 +1,93 @@
+// 1. Create a function that will get the sum of the numbers between 1 and n and return the answer
+const  suumation=function(N) {
+    let total = 0;
+      for(let i = 1; i <= N; i++){
+        total += i;
+      }
+      return total;
+  }
+
+  //2. Create a program to get the sum of all the even numbers in a group
+  const  suumEven=function(N) {
+    let total = 0;
+      for(let i = 2; i <= N; i+=2){
+        total += i;
+      }
+      return total;
+  }
+  // 3.Create a function to get the average of a group of numbers 
+  const avg=function([arr]) {
+    let totalAvg=0 ; 
+      for(let i=0;i<arr.length;i++) {
+          totalAvg+=arr[i] ; 
+          return totalAvg/arr.length ; 
+        
+      }
+
+  }
+// 4. Create a function to reverse the letters in a word
+const reverse=function(str) {
+    var newString = "";
+    for (let i = str.length - 1; i >= 0; i--) {
+        newString += str[i];
+    }
+    return newString;
+}
+reverse('hello');
+
+// 5. Create a function that takes an array of words and combines them with a dash
+const array=function([a]) {
+   console.log(a.join("-"))
+
+}
+array(['hi','my','test']) ; 
+
+// 6. Function that will count up to a number and back down and return a string of the climb
+
+
+
+
+// 7. Write a function that will tell you all of the words in an array that contain the letter `a`
+const wordsWithA=function([ar]) { 
+let newArray=[] ; 
+for(let i=0;i<ar.length;i++) { 
+  if(ar[i].search("a"))
```

Comment:
The `search()` function return the index of the searched item if it find a match, if it didn't find a match it will return `-1`, so your if statement condition should check if it's not equal to `-1`, like this `if(ar[i].search("a") !== -1)`.